apps/api/lib/api/web/controllers/player_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule API.Web.PlayerController do
+  use API.Web, :controller
+
+  action_fallback API.Web.FallbackController
+
+  def create(conn, %{"user_id" => user_id, "game_id" => game_id}) do
```

Comment:
We'll eventually want to do that bulk create so Kris can do his "select multiple to invite" feature and send all at once

Reply:
Ah really good point. I'm going to chnage that right now

Reply:
> chnage